model/Service/DataTypes/CountryCodeAlpha3.md

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# CountryCodeAlpha3
+
+## Summary
+
+A string constrained to the ISO 3166-1 alpha-3 3 character format.
+
+## Description
+
+The string must be in the [ISO 3166-1 alpha-3](https://www.iso.org/obp/ui/#iso:std:iso:3166:-3:ed-2:v1:en) format.  
```

Comment:
```suggestion
The string must be in the [ISO 3166-1 alpha-3](https://www.iso.org/obp/ui/#iso:std:iso:3166:-1) format.  
```

The current URL points to ISO 3166-3 which is "Part 3: Code for formerly used names of countries".

If a specific version of ISO 3166-1 is preferred, use https://www.iso.org/obp/ui/#iso:std:iso:3166:-1:ed-4:v1:en which is ISO 3166-1:2020 (latest).